src/io/flutter/module/settings/SettingsHelpForm.form

```diff
@@ -0,0 +1,138 @@
+            </constraints>
+            <properties>
+              <text value="Help..."/>
+            </properties>
+          </component>
+          <component id="f2329" class="javax.swing.JLabel" binding="projectTypeLabel">
+            <constraints>
+              <grid row="2" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <font style="1"/>
+              <text value="Project type:"/>
+            </properties>
+          </component>
+          <component id="4655f" class="javax.swing.JLabel" binding="projectTypeDescriptionForApp">
+            <constraints>
+              <grid row="2" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Help for app..."/>
+            </properties>
+          </component>
+          <component id="59377" class="javax.swing.JLabel" binding="orgLabel">
+            <constraints>
+              <grid row="4" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <font style="1"/>
+              <text value="Organization:"/>
+            </properties>
+          </component>
+          <component id="9e6c6" class="javax.swing.JLabel" binding="orgDescription">
+            <constraints>
+              <grid row="4" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Help..."/>
+            </properties>
+          </component>
+          <component id="1706b" class="javax.swing.JLabel" binding="projectTypeDescriptionForPlugin">
+            <constraints>
+              <grid row="3" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Help for plugin..."/>
+            </properties>
+          </component>
+          <grid id="a8d23" layout-manager="GridLayoutManager" row-count="2" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+            <margin top="5" left="0" bottom="0" right="0"/>
+            <constraints>
+              <grid row="0" column="1" row-span="1" col-span="1" vsize-policy="3" hsize-policy="3" anchor="0" fill="3" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties/>
+            <border type="none"/>
+            <children>
+              <component id="26e7" class="javax.swing.JTextPane" binding="gettingStartedUrl">
```

Comment:
I notice the styling of this looks a little different in your screenshot.  I wonder if using a LinkLabel (as we do for the SDK install link) would look more consistent?


![image](https://user-images.githubusercontent.com/67586/28623363-9e4fb8c0-71ca-11e7-9caf-161e45a578db.png)